pkg/apis/extensions/types.go

```diff
@@ -930,6 +930,16 @@ type PodSecurityPolicySpec struct {
 	// privilege escalation.
 	// +optional
 	AllowPrivilegeEscalation bool
+	// AllowedHostPaths is a white list of allowed host paths. Empty indicates that all host paths may be used.
+	// +optional
+	AllowedHostPaths []AllowedHostPath
+}
+
+// AllowedHostPath defines the host volume conditions that will be enabled by a policy
+// for pods to use. It requires the path to be defined.
+type AllowedHostPath struct {
+	// Path is the path prefix that the host volume must match
+	Path string
```

Comment:
It's confusing me that we use `path` anywhere but we mean "path prefix" really.

@jhorwit2 @pweil- @liggitt Maybe we should rename the fields? For example, it can be named `AllowedHostPathPrefixes`.